### PR TITLE
Display graph of loss through epochs

### PR DESCRIPTION
**Pull request by quentinverlhac**
Exemple on 3 epochs
PianocktailGru_000002

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -45,23 +45,25 @@ def generate_subspectrogram(duration_s=config.SUBSPECTROGRAM_DURATION_S, fft_rat
...
 
         for iteration, (spectro, label) in enumerate(train_dataset):
             predictions, labels = train_step(spectro, label, model, optimizer, train_loss, train_accuracy)
 
-            # display metrics
-            if iteration % 10 == 0:
-                utils.display_and_reset_metrics(train_loss, train_accuracy, predictions, labels, iteration=iteration)
+        # display metrics
+        loss_history.append(train_loss.result())
+        utils.display_and_reset_metrics(train_loss, train_accuracy, predictions, labels, epoch=epoch)
```
Comment (owain-Biddulph) [inline train.py]: Am I right in thinking this is now displayed every epoch ?

Comment (quentinverlhac) [reply train.py]: It is on purpose. I think it is enough to display it after every epoch
